specification/web/resource-manager/Microsoft.Web/stable/2018-02-01/CommonDefinitions.json

```diff
@@ -0,0 +1,2876 @@
+              "type": "string",
+              "readOnly": true
+            },
+            "possibleOutboundIpAddresses": {
+              "description": "List of IP addresses that the app uses for outbound connections (e.g. database access). Includes VIPs from all tenants. Read-only.",
+              "type": "string",
+              "readOnly": true
+            },
+            "containerSize": {
+              "format": "int32",
+              "description": "Size of the function container.",
+              "type": "integer"
+            },
+            "dailyMemoryTimeQuota": {
+              "format": "int32",
+              "description": "Maximum allowed daily memory-time quota (applicable on dynamic apps only).",
+              "type": "integer"
+            },
+            "suspendedTill": {
+              "format": "date-time",
+              "description": "App suspended till in case memory-time quota is exceeded.",
+              "type": "string",
+              "readOnly": true
+            },
+            "maxNumberOfWorkers": {
+              "format": "int32",
+              "description": "Maximum number of workers.\nThis only applies to Functions container.",
+              "type": "integer",
+              "readOnly": true
+            },
+            "cloningInfo": {
+              "$ref": "#/definitions/CloningInfo",
+              "description": "If specified during app creation, the app is cloned from a source app.",
+              "x-ms-mutability": [
+                "create"
+              ]
+            },
+            "snapshotInfo": {
+              "$ref": "#/definitions/SnapshotRecoveryRequest",
+              "description": "If specified during app creation, the app is created from a previous snapshot.",
+              "x-ms-mutability": [
+                "create"
+              ]
+            },
+            "resourceGroup": {
+              "description": "Name of the resource group the app belongs to. Read-only.",
+              "type": "string",
+              "readOnly": true
+            },
+            "isDefaultContainer": {
+              "description": "<code>true</code> if the app is a default container; otherwise, <code>false</code>.",
+              "type": "boolean",
+              "readOnly": true
+            },
+            "defaultHostName": {
+              "description": "Default hostname of the app. Read-only.",
+              "type": "string",
+              "readOnly": true
+            },
+            "slotSwapStatus": {
+              "$ref": "#/definitions/SlotSwapStatus",
+              "description": "Status of the last deployment slot swap operation.",
+              "readOnly": true
+            },
+            "httpsOnly": {
+              "description": "HttpsOnly: configures a web site to accept only https requests. Issues redirect for\nhttp requests",
+              "type": "boolean"
+            }
+          },
+          "x-ms-client-flatten": true
+        },
+      
```

Comment:
@ruslany `isLinux`? 😄 I don't know the details of this, but sounds like a year from now there could be an `isMacOS` or similar - wouldn't it make more sense to introduce an enum property that just says `platform` or something? May well be that that makes no sense here, just dumping my brain 🙂

Reply:
@olydis - while I agree that enum probably would be better, for now it is a bool. Should we add another platform later then we'll rework the API to be enum. As far as I know there are currently no plans for that.

Reply:
sounds good to me 🙂